src/components/organisms/NavBarSchedule/NavBarSchedule.scss

```diff
@@ -126,3 +134,28 @@ $schedule-btn--height: 51px;
     padding: $spacing--sm 0px;
   }
 }
+
+@media screen and (max-width: $media-breakpoint--md) {
+  .NavBarSchedule {
+    width: 100%;
```

Comment:
same as we have `$NavBarSchedule--width: 40%;`, let's create a variable for this property and use it 2 times: here and in the `.NavBarWrapper__download-buttons`